hangar-api/pom.xml

```diff
@@ -49,6 +49,12 @@
 			<version>19.0</version>
 		</dependency>
 
+        <dependency>
```

Comment:
hmm, indentation is off here - maybe I've used spaces rather than tabs? I could format the whole file if you like